### DISCUSSION TITLE
Hi there, got some question about Static & Hover Roll-up/down Portals

### DISCUSSION THREAD

**Discussion by RannerBoss** [Ross-Patterson/Portals-Desktop-Organization#35] [Q&A]
Hello, first thanks for the amazing tool!

Looking for some guidance, i was wondering about Portals Settings and if "Layouts" are "profiles"? that can coexist with other ones, as example: L1 has settings for static fixed Portals while L2 has the experimental Expand/Minimize Portal and hover/lost-focus ON. 
Also is there a way to do that in the current version? Just want to combine a couple of Static and hover Roll-up/down Portals on my desktop.

**Comment by Ross-Patterson** [comment]
Hi @RannerBoss,
At the moment, layouts only save the portals, their customization settings and positions. So what you are trying to achieve is not currently possible.

It would be theoretically possible to add a "Settings Profile" feature that allowed you to save and switch between settings profiles. I will add it to my backlog of things to investigate/consider.
